postgresql-11.yaml

```diff
@@ -115,8 +115,32 @@ subpackages:
           mkdir -p ${{targets.subpkgdir}}/opt/bitnami/postgresql
           mkdir -p ${{targets.subpkgdir}}/opt/bitnami/postgresql/conf
           mkdir -p ${{targets.subpkgdir}}/opt/bitnami/postgresql/bin
+          mkdir -p ${{targets.subpkgdir}}/opt/bitnami/postgresql/share
+
           ln -sf /usr/share/postgresql/pg_hba.conf.sample ${{targets.subpkgdir}}/opt/bitnami/postgresql/conf/pg_hba.conf
           ln -sf /usr/share/postgresql/postgresql.conf.sample ${{targets.subpkgdir}}/opt/bitnami/postgresql/conf/postgresql.conf
+
+          # Create symbolic links individually for each file
```

Comment:
Does it make sense to symlink wildcards? Or the whole directory?

```
ln -sf /usr/share/postgresql/ ${{targets.contextdir}}/opt/bitnami/postgresql/share
```

Reply:
I did that way because there are files which we created symlink by changing the names like:

```sh
          ln -sf /usr/share/postgresql/pg_hba.conf.sample ${{targets.subpkgdir}}/opt/bitnami/postgresql/conf/pg_hba.conf
          ln -sf /usr/share/postgresql/postgresql.conf.sample ${{targets.subpkgdir}}/opt/bitnami/postgresql/conf/postgresql.conf
```